util_inference_example.py

```diff
@@ -0,0 +1,87 @@
+"""
+Usage: python util_inference_example.py image annotations
+
+Adapted from the dense_inference.py to demonstate the usage of the util
+functions.
+"""
+
+import sys
+import numpy as np
+import cv2
+import densecrf as dcrf
+import matplotlib.pylab as plt
+from skimage.segmentation import relabel_sequential
+
+from utils import compute_unary, create_pairwise_bilateral, \
+    create_pairwise_gaussian
```

Comment:
Indeed the `create_pairwise_gaussian_2d` and `create_pairwise_bilateral_2d` are only for demonstration proposes and to verify/show that the more efficient mesh based implementations work correctly. I will mark them for now as private, but I'm also fine with deleting them completely.